apis/kueue/v1alpha1/multikueue_types.go

```diff
@@ -50,6 +50,14 @@ type KubeConfig struct {
 type MultiKueueClusterSpec struct {
 	// Information how to connect to the cluster.
 	KubeConfig KubeConfig `json:"kubeConfig"`
+
+	// A label value used to track the creator of workloads in the worker cluster.
+	//
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
+	// +kubebuilder:validation:MaxLength=63
+	// +kubebuilder:default=multikueue
+	Origin string `json:"origin,omitempty"`
```

Comment:
I see, we had an in-private discussion about this. This is introduced to allow multiple MultiKueue to reuse the same worker clusters by different instance of MultiKueue. Having configurable origin allows for that, however, with the default of `multikueue` this requires anyway a change in the config. I'm ok with the change, but also think it might be better to just say straight that this worker sharing is not supported. WDYT @alculquicondor @mwielgus ?

Reply:
I have discussed this with @mwielgus  in-private. We are ok keeping this configurable.